pages/agent/v3/aws/autoscaling_mac_metal.md.erb

```diff
@@ -0,0 +1,192 @@
+# Auto Scaling mac1.metal instances
+
+We have built a [CloudFormation template](https://github.com/buildkite/elastic-mac-for-aws)
+that configures an Auto Scaling group, Launch Template, and Host Resource Group
+suitable for maintaining a pool of EC2 mac1.metal instance based Buildkite
+Agents. These agents can be used to run Buildkite Pipelines that build Xcode
+based software projects for macOS, iOS, iPadOS, tvOS, and watchOS.
+
+As you must prepare and supply your own AMI for this template, macOS support has
+**not** been incorporated into the Elastic CI Stack for AWS.
+
+Using an Auto Scaling Group for your instances ensures booting your macOS
+Buildkite Agents is repeatable, and provides automatic instance replacement when
+hardware failures occur.
+
+## Prerequisites
+
+Familiarity with AWS VPCs, and EC2 AMIs is required. You should also have
+familiarity with the macOS GUI.
+
+You must choose an AWS Region with `mac1.metal` instances available. See
+[Amazon EC2 Mac Instances](https://aws.amazon.com/ec2/instance-types/mac/) and
+[Amazon EC2 Dedicated Hosts Pricing](https://aws.amazon.com/ec2/dedicated-hosts/pricing/)
+for details on which regions have `mac1.metal` hosts available.
+
+See also the [Amazon EC2 Mac instances User Guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-mac-instances.html)
+for more details on AWS EC2 Mac instances.
+
+## Choose a VPC Layout
+
+Before deploying this template you must choose a VPC subnet design, and which
+VPC security groups your instances should belong to.
+
+Depending on your threat model, you may find running instances in your default
+VPC’s public subnets with a public IP address suitable. Otherwise, you may wish
+to explore options like separate Public/Private subnets and a NAT Gateway, and
+using a Bastion or VPN to access the instances over SSH and VNC.
+
+In supported regions, `mac1.metal` dedicated hosts may not be available in
+every Availability Zone. Consider using a subnet in every Availability Zone to
+maximise the pool of instances available to boot from.
+
+You also need to configure or define the VPC Security Groups your instance
+network interfaces should belong to. At a minimum, inbound SSH access is
+required to set up your initial template AMI.
+
+## Build an AMI
+
+Before deploying this template, you must create a template AMI that can be
+horizontally scaled across multiple instances.
+
+1. Reserve a [AWS mac1.metal](https://aws.amazon.com/ec2/instance-types/mac/)
```

Comment:
I considered using a partial to share these instructions with the existing general AWS instructions, but they are sufficiently different I think having them separate makes sense.

Reply:
Definitely a situation where we should do the work so other folks don't have to.